tests/ManageCourses.Tests/UnitTesting/Client/ManageCoursesApiClientTest.cs

```diff
@@ -0,0 +1,190 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using FluentAssertions;
+using GovUk.Education.ManageCourses.ApiClient;
+using GovUk.Education.ManageCourses.Api.Model;
+using GovUk.Education.ManageCourses.Domain.Models;
+using NUnit.Framework;
+using Institution = GovUk.Education.ManageCourses.Domain.Models.Institution;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using System.Web;
+using Newtonsoft.Json;
+
+namespace GovUk.Education.ManageCourses.Tests.UnitTesting.Client
+{
+    [TestFixture]
+    public class ManageCoursesApiClientTest
+    {
+
+        private ManageCoursesApiClient sut;
```

Comment:
call it what it is. `manageCoursesApiClient`

Reply:
done, just for you :)